src/services/completions.ts

```diff
@@ -127,6 +128,27 @@ namespace ts.Completions {
         return { isGlobalCompletion: isInSnippetScope, isMemberCompletion: isMemberCompletionKind(completionKind), isNewIdentifierLocation, entries };
     }
 
+    const tsKeywords: string[] = [
+        SyntaxKind.EnumKeyword,
+        SyntaxKind.AbstractKeyword,
+        SyntaxKind.AsKeyword,

```

Comment:
I think these need to be context sensitive and need to be paired with different keyword filters. 
Eg. `as` is valid input `import * /*here*/as name from "module-name";`

Reply:
I think they are still context sensitive as usual, as showed in the test cases. the `as` keyword should be valid in JS, I removed it. And I updated the test cases to be more precise.